lib/settings.py

```diff
@@ -110,7 +116,7 @@ def HaveChanged(importSettings: xbmcaddon.Settings, providerSettings: xbmcaddon.
             SynchronizationSettings.SaveHash(importSettings, newHash)
 
         return True
-    
+
```

Comment:
This change looks unrelated. Can you move it into a separate "cosmetics" commit?